setup.py

```diff
@@ -49,7 +49,7 @@
     install_requires=[
         "eval_type_backport>=0.1.0; python_version < '3.10'",
         "pydantic>=2.6.0",
-        "requests>=2.20",
+        "requests>=2.23",
```

Comment:
Is this the lowest version we can set it at? Was `2.20` not working well?

Reply:
It didn't cause any problems, but would never be installed anyway because `2.23` was the first that included python 3.8. `2.23` is what we have pinned in `web3` for the same reason.

Reply:
Even this might be false advertising, because no test run we do is going to go through and install the bottom pins of everything just to make sure.

Reply:
> 2.23 was the first that included python 3.8

That's the reason I was looking for, thanks. I tried it with `2.23` and it works just fine.